docs/root/operations/admin.rst

```diff
@@ -184,15 +184,24 @@ The fields are:
   Histograms will output the computed quantiles i.e P0,P25,P50,P75,P90,P99,P99.9 and P100.
   The output for each quantile will be in the form of (interval,cumulative) where interval value
   represents the summary since last flush interval and cumulative value represents the
-  summary since the start of envoy instance.
+  summary since the start of envoy instance. "No recorded values" in the histogram output indicates
+  that it has not been updated with a value.
   See :ref:`here <operations_stats>` for more information.
 
+  .. http:get:: /stats?usedonly
+
+  Outputs statistics that have been updated with values on demand.
+
+  .. http:get:: /stats?format=json&usedonly
```

Comment:
nit: move this below the json documentation?

Reply:
Yes. Makes sense. Actually /stats?format=json has wrong alignment. I have moved it to the right place and moved this inside that. So /stats, /stats?format=json, /stats?format=prometheus are at the same level and usedonly will come inside them.